### PR TITLE
Add additional interactive elements to the home page's challenge sample

### PR DESCRIPTION
**Pull request by TheComputerNerd88**
We show realistic alerts on the home page when the challenge sample "submit flag" buttons are pressed, demonstrating to users how the flag submission system works. Additionally, the challenge sample filter tags are now clickable and provide the user feedback with a highlight identical to that on the challenge page.

### CODE REVIEW

File: static/js/index.js
Comment (Alexolotal) [inline static/js/index.js:1]: Maybe the alert should be clear that you *would have* "earned 20 points", as given that this alert is basically identical to an actual solve, I'm worried someone doing their first CTF may get confused

Comment (TheComputerNerd88) [reply static/js/index.js:1]: Fair - originally I wanted this alert to be _as similar_ as possible, which is why I kept all the text the same as if it were real, to have the highest degree of immersion.

Comment (TheComputerNerd88) [reply static/js/index.js:1]: image
This seems a bit clunky.

Comment (Alexolotal) [reply static/js/index.js:1]: maybe greying out the alert a little to indicate its an example? or making the footer more prominent.

Comment (TheComputerNerd88) [reply static/js/index.js:1]: image
Does this work? I think it's a nice balance between a realistic alert and denoting that it's an example.

Comment (Alexolotal) [reply static/js/index.js:1]: Yea that looks good, the bold example alert does make it a bit more prominent

**Review by Alexolotal — APPROVED** [review]

**Pull request merged**